apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/config/KeystoneAuthenticationModule.java

```diff
@@ -234,6 +234,13 @@ public String get() {
       return authenticationMethods.get(credentialType);
    }
 
+   @Provides
+   @Singleton
+   @Named("credentialsRetryable")
+   protected final boolean credentialsRetryable(Function<Credentials, Access> getAccess) {
```

Comment:
@nacx You're right. I've updated the pull request following your advice (injecting `CredentialType`).